### PR TITLE
Enforce RuboCop and correct existing syntax

### PR DESCRIPTION
**Pull request by tarcieri**
This enables RuboCop by default and fixes the existing offences

### CODE REVIEW

File: lib/rbnacl/boxes/curve25519xsalsa20poly1305.rb [hunk 1]
```diff
@@ -64,14 +64,14 @@ module Boxes
...
-      sodium_constant  :BOXZEROBYTES
-      sodium_constant  :BEFORENMBYTES
-      sodium_constant  :PUBLICKEYBYTES
-      sodium_constant  :SECRETKEYBYTES, :PRIVATEKEYBYTES
+      sodium_constant :NONCEBYTES
+      sodium_constant :ZEROBYTES
+      sodium_constant :BOXZEROBYTES
+      sodium_constant :BEFORENMBYTES
+      sodium_constant :PUBLICKEYBYTES
+      sodium_constant :SECRETKEYBYTES, :PRIVATEKEYBYTES
```
Comment (namelessjon) [inline lib/rbnacl/boxes/curve25519xsalsa20poly1305.rb]: Stuff like this with rubocop really bugs me.  I think version where they're all aligned to the same vertical is a lot more readable.

**Comment by namelessjon** [comment]
A lot of it makes sense, but some of the stylist choices bug me, and I don't think they help with the readability.

On the other hand, being able to push toward shorter methods is good.

File: lib/rbnacl/boxes/curve25519xsalsa20poly1305.rb (hunk 1 above)
Comment (tarcieri) [inline lib/rbnacl/boxes/curve25519xsalsa20poly1305.rb]: RuboCop is configurable. We can shut this particular one off. And FWIW I agree. This patch brings us close to in-line with the RuboCop defaults, so... let the bikeshedding begin! :wink:

Comment (tarcieri) [inline lib/rbnacl/boxes/curve25519xsalsa20poly1305.rb]: @namelessjon I disabled the SingleSpaceBeforeFirstArg cop and put these back to what they were. Any other objections?

Comment (namelessjon) [inline lib/rbnacl/boxes/curve25519xsalsa20poly1305.rb]: No, I think the rest is all good!

**Comment by tarcieri** [comment]
If you have no other objections, merge it? :wink:

**Comment by namelessjon** [comment]
Done

**Pull request merged**